rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/api/JaxRsActivator.java

```diff
@@ -10,7 +10,7 @@
 // mark the war as a JAX-RS archive
 @ApplicationPath("/")
 @OpenAPIDefinition(
-    info = @Info(title = "LRA Coordinator", version = "1.0"),
+    info = @Info(title = "LRA Coordinator", version = "1.0-RC1"),
```

Comment:
This looks like something we will have to watch out for. I wonder if how we can make this more prominent that it needs to be updated with LRA coordinator versions?

Reply:
I appreciate this concern is not introduced by this PR but is something to follow up on

Reply:
I created a final constant (in the same file) to give some degree of prominence to it. Now that you have mentioned it I thing I will remember to update it as we produce new LRA releases.